book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+## What is cultural change?
+
+Culture is the social environment in which you partake. 
+Culture has an influence on your views, values, concerns and your identity. 
+Culture can be difficult to change as it is hard to address directly and shared values can be deeply ingrained ({cite:ps}`Kotter2012change`). 
+Instead, norms of behaviour may be easier to change. 
+
+Here, cultural change is defined as “inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation"
```

Comment:
Is there a citation that goes here?

Reply:
Eh.. Plomp 2022? 😄 Maybe I should remove the quotation marks?